src/irc/IdentGenerator.ts

```diff
@@ -208,6 +177,31 @@ export class IdentGenerator {
         return uname;
     }
 
+    /**
+     * Get the next available suffix for a given complete username.
+     * This function will find the correct suffix for a user in as fewest
+     * DB calls as possible.
+     * @param uname The IRC username
+     * @param domain The IRC domain.
+     */
+    private async getSuffixForUsername(username: string, domain: string) {
+        let suffixLength = 1;
+        while (suffixLength < IdentGenerator.MAX_USER_NAME_SUFFIX.toString().length) {
```

Comment:
Log10 tells you how many zeros there are after the 1. 😏
```suggestion
        while (suffixLength < Math.log10(IdentGenerator.MAX_USER_NAME_SUFFIX) + 1) {
```
![giphub](https://media1.giphy.com/media/BmmfETghGOPrW/giphy.gif)

Fell free to ignore this as yours is more readable.